payloadinterceptor.go

```diff
@@ -41,26 +35,12 @@ func (s *streamingClientInterceptor) Send(msg any) error {
 }
 
 func (s *streamingClientInterceptor) CloseRequest() error {
```

Comment:
Since `StreamingClientConn` is embedded (and this method would automatically be delegated to the embedded value), you can simply remove this, right?